deploy_kubernetes.md

```diff
@@ -0,0 +1,77 @@
+# website-monitoring deployed on kubernetes
+
+## Dependencies
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/)
+* [kompose](https://kompose.io/installation/)
+
+## Usage
+
+* `git clone git@github.com:mbelloiseau/website-monitoring.git && cd website-monitoring`
+* Edit `config/prometheus/targets.yml` (see targets.yml.example) or use `./gen_target.sh website-1.tld website-2.tld ...`
+* Convert resources `kompose convert -f docker-compose.yml -o website-monitoring-deploy-k8s.yaml`
```

Comment:
This will require an understanding of kubernetes resources for the original docker-compose users, which is fine if you assume that all subsequent users will be kubernetes users.

Reply:
I think we can be sure about this point.

Reply:
got